nav2_controller/src/controller_server.cpp

```diff
@@ -630,14 +634,25 @@ void ControllerServer::computeAndPublishVelocity()
 
   geometry_msgs::msg::Twist twist = getThresholdedTwist(odom_sub_->getRawTwist());
 
+  auto transformed_plan = path_handler_->transformGlobalPlan(
+    pose, max_robot_pose_search_dist_, interpolate_curvature_after_goal_);
+  // RCLCPP_INFO(get_logger(), "compute remaining distance %lf ",nav2_util::geometry_utils::calculate_path_length(transformed_plan));
+  global_path_pub_->publish(transformed_plan);
```

Comment:
I think if this is pruned, in the odom frame, and such, this is a local plan

Reply:
Also, make this into a unique pointer `make_unique<nav_msgs::msg::Path>(local_path_)`